Gemfile

```diff
@@ -50,15 +50,18 @@ group :development, :test do
   gem 'brakeman', require: false # A static analysis security vulnerability scanner for Ruby on Rails applications
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/nimblehq/nimble-survey-web/blob/e2e4612e1a042effe4fb416e743d1c929888c14f/Gemfile#L50"></span>Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem <code>brakeman</code> should appear before <code>sassc-rails</code>.</td>
    </tr>
  </tbody>
</table>